context-js/tests/parse_and_validate.test.ts

```diff
@@ -80,6 +81,7 @@ describe("context-js", () => {
 
     let parse_result = junit_parse(Buffer.from(validJunitXml, "utf-8"));
     let report = parse_result.report;
+    assert(report, "Report from parsing report should not be null");
```

Comment:
Why not use `expect`?

Reply:
`expect` doesn't type narrow like `assert` does so it was just a way to not have to slap a `?` on everything